Source/WebKit/UIProcess/mac/WKTextAnimationManager.mm

```diff
@@ -124,10 +129,13 @@ - (void)addTextAnimationForAnimationID:(NSUUID *)uuid withData:(const WebCore::T
                 strongWebView->page().updateUnderlyingTextVisibilityForTextAnimationID(remainingID, false);
         }).get();
 
-        effect.get().completion = makeBlockPtr([weakWebView = WeakPtr<WebKit::WebViewImpl>(_webView), remainingID = data.unanimatedRangeUUID, uuid = RetainPtr(uuid), runMode = data.runMode] {
+        effect.get().completion = makeBlockPtr([weakSelf = WeakObjCPtr<WKTextAnimationManager>(self), weakWebView = WeakPtr<WebKit::WebViewImpl>(_webView), remainingID = data.unanimatedRangeUUID, uuid = RetainPtr(uuid), runMode = data.runMode] {
```

Comment:
when do we use `retainPtr` vs `RetainPtr`? It seems the former is more frequently used in lambda captures?

Reply:
I don't think there's an established convention? This file uses `RetainPtr` a bunch so I just matched it.

Though, I'd be in favor of moving everything over to `retainPtr` at some point since it's more commonly used.